src/poetry/core/masonry/builders/wheel.py

```diff
@@ -216,6 +216,15 @@ def _copy_file_scripts(self, wheel: zipfile.ZipFile) -> None:
             )
 
     def _run_build_command(self, setup: Path) -> None:
+        if self._editable:
+            subprocess.check_call(
+                [
+                    self.executable.as_posix(),
+                    str(setup),
+                    "build_ext",
+                    "--inplace",
+                ]
+            )
         subprocess.check_call(
```

Comment:
is it necessary / useful also to call `setup.py build` in the editable case, or was `setup.by build_ext --inplace` enough?

Reply:
not sure about any of that.  since this code is only executed for packages with a build script, poetry surely can't be relying on it do deal with whatever `build_scripts` does?  (Probably it's equivalent to poetry's `WheelBuilder._copy_file_scripts()`?)

maybe what we are learning here is that it should have just been `build_ext` all along; possibly `--in-place` for editable intalls.

Reply:
I'm inclined to agree with you about `build_scripts`; I took a closer look and I don't think there's anything that poetry needs out of it. I've never encountered `build_clib`, so I'm not sure what the usage pattern would be. It doesn't support a build inplace flag, so I assume it would be enough to just call it as a separate command. If preferred, I'd be happy to modify this to e.g.:

```python
        build_ext_cmd = [
            self.executable.as_posix(),
            str(setup),
            "build_ext",
        ]
        if self._editable:
            build_ext_cmd.append("--inplace")
        subprocess.check_call(build_ext_cmd)

        # Optionally add:
        subprocess.check_call(
            [
                self.executable.as_posix(),
                str(setup),
                "build_clib",
            ]
        )
```

Reply:
I don't think `build_clib` is relevant either, that looks as though it deals with `libraries` in setup.py, and poetry's generated `setup.py` does not include that.

So yeah, the first half of that is what I think this probably ought to look like; so long as it doesn't seem to break anything...

Reply:
I think if it were to be used, `libraries` would get injected into setup_kwargs inside build.py similar to `ext_modules`. 

I'm happy to drop it if it's not something the maintainers want to support, just don't want to inadvertently break anyone's use case haha

Reply:
> libraries would get injected into setup_kwargs inside build.py similar to ext_modules.

ah, perhaps that's right.  

don't think I mind much either way then.  `build_ext` followed by `build` still feels logically wasteful; but `build_ext` followed by `build_clib` isn't such an obvious win

Reply:
Sounds good, I think I'll stick with `build_ext` followed by `build` in that case, it does feel a little more wasteful, but also might be a little less fragile in the (perhaps unlikely) event that another useful subcommand is added to `build`.